external/.gitignore

```diff
@@ -3,6 +3,7 @@ aarch64-linux-gnu
 arm-linux-gnueabihf
 x86_64-pc-linux-gnu
 arm64-apple-darwin*
+x86_64-apple-darwin*
```

Comment:
This change was merged in #5710, but it doesn't appear to cause a merge conflict, so I think it's okay.